### PR TITLE
Owncloud

### PR DESCRIPTION
**Pull request by laurentperrinet**
upgrading to the latest stable version

### CODE REVIEW

File: Casks/owncloud.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 cask :v1 => 'owncloud' do
-  version '1.7.1.1655'
-  sha256 '31cff973611823b5306544fe394539c4bb167645fb14622125443967f20014d4'
-
```
Comment (vitorgalvao) [inline Casks/owncloud.rb]: Please leave this empty line there. It’s used to visually separate what needs to be updated every time from what doesn’t.

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by meduz** [comment]
readded the missing empty line but failed to squash...

**Comment by vitorgalvao** [comment]
No worries. It will be merged in [a different pull request](https://github.com/caskroom/homebrew-cask/pull/10094) squashed. Your contribution is still included (and still credited to you). Thank you for the contribution

**Pull request closed without merging**